src/azul/indexer/document.py

```diff
@@ -481,7 +481,7 @@ def to_json(self) -> JSON:
                     contents=self.contents)
 
     def to_dict(self) -> JSON:
-        return {f.name: getattr(self, f.name) for f in fields(self)}
+        return {f.spec: getattr(self, f.spec) for f in fields(self)}
```

Comment:
Extremely worried why we didn't catch this either. Please find out why.

Reply:
delete as first commit and run tests

Reply:
Unit tests pass + IT passes with this method removed